.github/workflows/ci.yml

```
@@ -99,6 +99,11 @@ jobs:
             arch: arm64-v8a
             api: 33
           type: { name: Debug, flags: -DCMAKE_BUILD_TYPE=Debug -DSFML_FATAL_OPENGL_ERRORS=ON }
+        exclude:
```

I'd add a comment to explain the exclusion

```suggestion
        exclude: # Shared libraries are not supported on iOS
```